<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ValidationDialog</class>
 <widget class="QDialog" name="ValidationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QWidget" name="dialogContainer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>684</width>
       <height>362</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>684</width>
       <height>362</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#dialogContainer {
	border-radius: 12px;
	background-color: white;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>18</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>18</number>
         </property>
         <property name="leftMargin">
          <number>18</number>
         </property>
         <property name="topMargin">
          <number>16</number>
         </property>
         <property name="rightMargin">
          <number>18</number>
         </property>
         <property name="bottomMargin">
          <number>16</number>
         </property>
         <item>
          <widget class="QLabel" name="dialogTitle">
           <property name="styleSheet">
            <string notr="true">color: black;
font-family: Open Sans;
font-weight: 500;
font-size: 22px;
line-height: 30px;
letter-spacing: 0%;
</string>
           </property>
           <property name="text">
            <string>Validation Error</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="closeButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../../Resources/Resources.qrc">
             <normaloff>:/images/Project/cross-icon.svg</normaloff>:/images/Project/cross-icon.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>8</number>
         </property>
         <property name="leftMargin">
          <number>18</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>18</number>
         </property>
         <property name="bottomMargin">
          <number>32</number>
         </property>
         <item alignment="Qt::AlignmentFlag::AlignHCenter">
          <widget class="QWidget" name="iconWidget" native="true">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>88</width>
             <height>88</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>88</width>
             <height>88</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LayoutDirection::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background-image: url(:/images/Project/warning-icon.svg);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="confirmTitleLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>44</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>44</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #1A1A1A;
font-family: Open Sans;
font-weight: 600;
font-size: 26px;
line-height: 44px;
letter-spacing: 0%;
text-align: center;
</string>
           </property>
           <property name="text">
            <string>Validation Error</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="messageLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #4D4D4D;
font-family: Open Sans;
font-weight: 500;
font-size: 20px;
line-height: 34px;
letter-spacing: 0%;
text-align: center;
</string>
           </property>
           <property name="text">
            <string>Please correct the errors and try again.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="bottomGroup" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>42</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>16</number>
         </property>
         <property name="leftMargin">
          <number>18</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>18</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="okButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>42</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background-color: #FCCA69;
	color: white;
	font-family: Open Sans;
	font-weight: 500;
	font-size: 18px;
	line-height: 24px;
	letter-spacing: 0%;
	border-radius: 8px;
}
</string>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../Resources/Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ValidationDialog</receiver>
   <slot>onCloseButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>751</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ValidationDialog</receiver>
   <slot>onOkButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onOkButtonClicked()</slot>
  <slot>onCloseButtonClicked()</slot>
 </slots>
</ui>
